.github/workflows/javadoc.yml

```diff
@@ -0,0 +1,28 @@
+# This workflow builds the Javadoc
+name: Javadoc
```

Comment:
Don't we need to upload the generated artifacts? 
I guess they should also be published on Maven Central.

Reply:
We store these artifacts in Java CI:

```
            build/reports/
            */build/test-results/**/*.xml
```

None of these contain the generated Javadoc.
Is there actually anything that we use from the build for publication?

Reply:
At least the snapshot repo has Javadoc jars with content.

![image](https://user-images.githubusercontent.com/43337019/150168944-fefbf915-08a9-4cc3-a587-cdb8a0a9a3ff.png)

Reply:
You're right. We seem to build all artifacts for publication in that specific workflow.